mmdeploy/utils/timer.py

```diff
@@ -127,3 +129,32 @@ def activate(cls,
         else:
             for name in cls.names:
                 cls.names[name]['enable'] = False
+
+    @classmethod
+    def print_stats(cls, name: str):
+        """print statistics results of timer.
+
+        Args:
+            name (str): The name registered with `count_time`.
+        """
+        from prettytable import PrettyTable
+
+        assert name in cls.names
+        stats = cls.names[name]
+        execute_time = stats['execute_time']
+        latency_mean = 1000 * float(np.mean(execute_time))
+        latency_median = 1000 * float(np.median(execute_time))
+        latency_min = 1000 * float(np.min(execute_time))
+        latency_max = 1000 * float(np.max(execute_time))
+        fps_mean, fps_median = 1000 / latency_mean, 1000 / latency_median
+        fps_min, fps_max = 1000 / latency_min, 1000 / latency_max
+        results = PrettyTable()
+        results.field_names = ['Stats', 'Latency/ms', 'FPS']
+        results.add_rows([
+            ['Mean', latency_mean, fps_mean],
+            ['Median', latency_median, fps_median],
+            ['Min', latency_min, fps_min],
+            ['Max', latency_max, fps_max],
+        ])
+        results.float_format = '.3'
+        print(results)
```

Comment:
Logging methods of mmcv and mmdet prefer not using `print`

Reply:
print is ok with me. Same as mmdet: https://github.com/open-mmlab/mmdetection/blob/56e42e72cdf516bebb676e586f408b98f854d84c/tools/test.py#L268